tests/sc_dbstore_func.test/output.expected

```diff
@@ -0,0 +1,16 @@
+- Testing dbstore now() -
+1
+-- The schema change below should fail --
+[ALTER TABLE t1 { tag ondisk { int i datetime t dbstore={now()} } }] failed with rc 240 cannot add new field without dbstore or null
```

Comment:
This error message is confusing because you have dbstore here

Reply:
maybe add nullable dbstore function here?

Reply:
Thanks for the suggestion! I've reworded this to "column must be nullable to use a function as its default value". Does it sound better?

Reply:
Yep, thanks!